backend/src/utils/file_processor_utils.py

```
@@ -14,7 +14,7 @@
 
 from models.transaction_file import FileFormat, ProcessingStatus
 from models.file_map import FileMap
-from utils.transaction_parser import parse_transactions
+from utils.transaction_parser_new    import parse_transactions
```

Ruff (0.11.9)

`utils.transaction_parser_new.parse_transactions` imported but unused

Remove unused import: `utils.transaction_parser_new.parse_transactions`

(F401)